app/mailers/projet_mailer.rb

```diff
@@ -25,6 +26,7 @@ def notification_invitation_intervenant(invitation)
     @invitation = invitation
     mail(
       to: invitation.projet.email,
+      cc: invitation.projet.personne.try(:email),
```

Comment:
J'y ai pensé mais ça complexifie la chose : il faut la mettre en `to:` si le projet n'a pas d'email.
J'ai vérifié que le mail était envoyé si personne n'était en `to:` avec une personne en `cc:` et le mail est bien envoyé.